client/src/pages/Restaurant/RestaurantMap/index.js

```diff
@@ -0,0 +1,77 @@
+/*global google*/
+import React, { useState } from "react";
+import { GoogleMap } from "@react-google-maps/api";
+import RestaurantMarker from "./RestaurantMarker";
+import UserMarker from "./UserMarker";
+import getUserCoordinates from "../../../utils/getUserCoordinates";
+import computeMidpoint from "../../../utils/computeMidpoint";
+
+const containerStyle = {
+  width: "100%",
+  height: "30vh",
+};
+
+function Map({ restaurantLatitude, restaurantLongitude }) {
+  const [map, setMap] = React.useState(null);
+  const [userCoordinates, setUserCoordinates] = useState([]);
+
+  const mapOptions = {
+    zoomControlOptions: {
+      position: google.maps.ControlPosition.RIGHT_TOP,
+    },
+    mapTypeControl: false,
+    streetViewControl: false,
+  };
+
+  React.useEffect(() => {
+    if (!map) return;
+    getUserCoordinates()
+      .then((result) => {
+        setUserCoordinates(result.coordinates);
```

Comment:
dont need this. just make a variable called `userCoordinates`. This state isnt being used anywhere else.